### ISSUE TITLE
Cannot set where $not inside @Scopes declaration

### ISSUE THREAD

**Issue by micksatana** [RobinBuschmann/sequelize-typescript#137]
Model file:

```
@DefaultScope({
    attributes: [
        'ctn_id',
        'ctn_code'
    ]
})
@Scopes({
    test1: {
        where: {
            ctn_code: {
                $not: null
            }
        }
    }
})
@Table({
    tableName: 'contents'
})
export default class Content extends Model<Content> {
    @PrimaryKey
    @AutoIncrement
    @Column({type: DataType.INTEGER(10).UNSIGNED})
    readonly ctn_id: number;

    @Column({type: DataType.STRING(255)})
    ctn_code: string;
}
```

When defining where condition with `$not: null` or `$ne: null` in the @Scopes, it will cause an error.

```
TypeError: Cannot convert undefined or null to object
      at node_modules/sequelize-typescript/lib/utils/object.js:10:14
      .....
```

**Comment by RobinBuschmann** [comment]
@micksatana This should be fixed with `sequelize-typescript@0.5.0` :)

**Comment by swist** [comment]
had the same problem, can confirm that it works as of `0.5.0`
